vmdk_plugin/plugin.go

```diff
@@ -117,19 +116,53 @@ func (d *vmdkDriver) mountVolume(name string) (string, error) {
 		return mountpoint, err
 	}
 
+	if d.useMockEsx {
+		return mountpoint, fmt.Errorf("No device to mount.")
+	}
+
+	watcher, err := inotify.NewWatcher()
+	if err != nil {
+		log.WithFields(
+			log.Fields{"name": name, "dir": mountpoint},
+		).Error("Failed to create watcher ")
+		return mountpoint, err
```

Comment:
Watcher errors should not result in mountVolume error. We should log & proceed to check if disk is mounted anyway.